jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBStatement.java

```diff
@@ -215,8 +215,8 @@ public boolean execute(String arg0, String[] arg1) throws SQLException {
 
   /**
    * There are two kinds of sql here: (1) query sql (2) update sql.
-   * (1) return IoTDBJDBCResultSet or IoTDBNonAlignJDBCResultSet
-   * (2) simply get executed
+   * <p>
+   * (1) return IoTDBJDBCResultSet or IoTDBNonAlignJDBCResultSet (2) simply get executed
```

Comment:
In this way, the format will not be modified when the shortcut keys are used in the idea to automatically format, otherwise, the format will be modified if the shortcut keys are used.